lightcurve/templates/lightcurve.html

```diff
@@ -18,12 +22,17 @@
 .image-container {
   margin: 2em 0;
 }
+
 </style>
 {% endblock %}
 
 {% block heading %}
-  Light Curve <span>{{lightcurve.name}}</span>
-  <a href="{% url "upload_image" %}/{{lightcurve.id}}" class="btn btn-primary pull-right">Add New Images</a>
+  <form id="lightcurve_{{lightcurve.id}}_title" onsubmit="saveLightcurveChanges({{lightcurve.id}}); return false" method="post">
```

Comment:
I think we can simplify the code by omitting the lightcurve id from the form id, because there is only one per lightcurve page.

But in general, using class is preferred because it allows for multiple uses on the page if that's something we eventually want to do.  In most cases, the rest of the code follows a convention where CSS selectors used in js are typically classes prefixed by `js-`.  For example, `<form class="js-lightcurve-title">`.  This makes it clearer which classes are used in scripts vs design.

Reply:
Done. Lightcurve id removed.